### PR TITLE
#163369300 Refactor and Implement code changes based on feedback

### PR DESCRIPTION
**Pull request by tirgei**
#### What does this PR do?
This Pull Request implements feeback from the LFA

#### Description of Task to be completed?
The tasks for this PR is:
- Refactor and move tests to the root folder so that the test files are not tested too
- Remove access_token and refresh_token from signup response
- Add validations for create meetup so that meetup with same details is not created again

#### How should this be manually tested?
- Clone the repo

```console
foo@bar:~$ git clone https://github.com/tirgei/Questioner-API.git
```

- Create the psql databases

```console
foo@bar:~$ createdb questioner_db
foo@bar:~$ createdb questioner_test_db
```

- CD into the folder

```console
foo@bar:~$ cd Questioner-API
```

- Create a virtual environment

```console
foo@bar:~$ python3 -m venv env
```

- Activate the virtual environment

```console
foo@bar:~$ source env/bin/activate
```

- Install dependencies

```console
foo@bar:~$ pip install -r requirements.txt
```

- Set the environment variables

```console
foo@bar:~$ mv .env.example .env
foo@bar:~$ source .env
```

- Create database tables

```console
foo@bar:~$ python manage.py -a
```

- Run the tests

```console
foo@bar:~$ pytest --cov=app
```

- Run the app

```console
foo@bar:~$ flask run
```

#### Pivotal Tracker story associated with this PR
[#163369300](https://www.pivotaltracker.com/story/show/163369300)

#### Screenshots
Refactor and move tests and make sure they are okay
feedback-refactor-tests

Remove access_token and refresh_token from signup response
refactor-signup-response

Create meetup at the same day with same topic
feedback-same-date-topic

Create meetup at the same day at the same venue
feedback-same-date-venue

Create meetup with same topic at the same venue
feedback-same-topic-venue

### CODE REVIEW

File: app/api/models/meetup_model.py [hunk 1]
```diff
@@ -111,4 +111,34 @@ def update_tags(self, meetup_id, meetup_tags):
         self.conn.commit()
         return self.cur.fetchone()
 
+    def collides(self, data):
+        """ Check if meetup collides with another meetup """
```
Comment (hoslack) [inline app/api/models/meetup_model.py]: Hello @tirgei , Great job on the changes. But this name `collides`, for a function, brings a lot of confusion. Please rename it to something that suggests that there is a duplicate.

**Review by hoslack — CHANGES_REQUESTED** [review]

**Review by hoslack — CHANGES_REQUESTED** [review]
Hello @tirgei The name of your PR does not follow the guidelines. Please read this [Andela best practices](https://github.com/andela/bestpractices/wiki)

**Comment by colMike** [comment]
Really clean code...Awesome work

File: app/api/models/meetup_model.py (hunk 1 above)
Comment (tirgei) [reply app/api/models/meetup_model.py]: Noted

**Comment by tirgei** [comment]
> Hello @tirgei The name of your PR does not follow the guidelines. Please read this [Andela best practices](https://github.com/andela/bestpractices/wiki)

Noted

**Comment by tirgei** [comment]
> Really clean code...Awesome work

Thanks

**Pull request merged**